undertow/src/main/java/org/wildfly/extension/undertow/UndertowTransformers.java

```diff
@@ -125,6 +127,29 @@ private static void registerTransformersWildFly18(ResourceTransformationDescript
                 .getAttributeBuilder()
                     .setValueConverter(AttributeConverter.DEFAULT_VALUE, CONNECTION_IDLE_TIMEOUT)
                 .end();
+
+        final ResourceTransformationDescriptionBuilder serverBuilder = subsystemBuilder.addChildResource(UndertowExtension.SERVER_PATH);
+
+        final AttributeTransformationDescriptionBuilder http = serverBuilder.addChildResource(UndertowExtension.HTTP_LISTENER_PATH).getAttributeBuilder()
+                .setDiscard(DiscardAttributeChecker.DEFAULT_VALUE, WRITE_TIMEOUT)
```

Comment:
I need to check this with @bstansberry  or someone else that has more knowledge on transformers than I do.
I think that, because we are just adding a default value to an already existing attribute, we don't add any rejects here. I think we just need to:
- either add the default value when transforming to older versions, so that the absence of a read/write-timeout in current WF model means a read/write-timeout value set to 60 in previous versions
- or we just omit the value if it is undefined (meaning it will have the default value of 60) when transforming to older versions, so that the transformed model is clean of that
I would bet on the first option, but I could be wrong. As I said, we need some feedback from transformers pros :-)

Reply:
This is a quick, perhaps off-topic comment before I go into meetings.

https://docs.wildfly.org/25/Extending_WildFly.html#common-transformation-use-cases covers some of the cases; perhaps this is 4.8.3?

The basic requirement is that servers running on the legacy version need to end up with the same runtime behavior as those running the current version, or the transformer logic should result in the op setting the config to whatever the legacy server can't handle being rejected.

Reply:
The use of DiscardAttributeChecker.DEFAULT_VALUE here looks correct to me.